Mathlib/Topology/Constructible.lean

```diff
@@ -296,10 +297,13 @@ variable [CompactSpace X] {P : ∀ s : Set X, IsConstructible s → Prop} {B : S
 lemma _root_.IsRetrocompact.isCompact (hs : IsRetrocompact s) : IsCompact s := by
   simpa using hs CompactSpace.isCompact_univ
 
+variable [QuasiSeparatedSpace X]
+
 /-- Variant of `TopologicalSpace.IsTopologicalBasis.isRetrocompact_iff_isCompact` for a
 non-indexed topological basis. -/
+@[stacks 0069 "Iff form of (2). Note that Stacks doesn't define quasi-separated spaces."]
 lemma _root_.TopologicalSpace.IsTopologicalBasis.isRetrocompact_iff_isCompact'
-    (basis : IsTopologicalBasis B) (compact_inter : ∀ U ∈ B, ∀ V ∈ B, IsCompact (U ∩ V))
```

Comment:
Can you add a constructor of `QuasiSeparatedSpace X` from `(basis : IsTopologicalBasis B) (compact_inter : ∀ U ∈ B, ∀ V ∈ B, IsCompact (U ∩ V))`? Thanks!